workbench-core/datasets/src/schemas/createDataSet.ts

```diff
@@ -11,13 +11,11 @@ const CreateDataSetSchema: Schema = {
   type: 'object',
```

Comment:
Has there been discussion about moving the schema objects into `swb-app` since they are SWB specific?

Reply:
Not yet. @bhudnell , you will be the part of those discussions for sure

Reply:
Resolving as it's being discussed offline